cmd/argocd/commands/admin/settings_rbac.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/gobwas/glob"
```

Comment:
Is there a particular reason you did use gobwas/glob directly, instead of util/glob provided by the argo-cd package?

Reply:
Nope, I didn't realize there was a util/glob provided by the argo-cd package. I have updated the code to use util/glob instead of gobwas/glob directly.